platformio.ini

```diff
@@ -396,6 +397,7 @@ build_flags =
     ${esp32git.build_flags}
     -D BOARD_NAME='"esp32_devkitc"'
     -D BOARD_ESP32_DEVKITC
+    -D USE_OLED
```

Comment:
Please remove this. This is a global setting. Please add the #define in an appropriate gpio def file or make a variant for your HW platform.